GITHUB_ORG_MANGEMENT_POLICY.md

```diff
@@ -89,4 +89,20 @@ Bots that perform actions on behalf of the project (such as moderation or member
 management actions) are required to maintain a log, accessible to all individuals
 granted Owner permissions, of all actions taken.
 
+## GitHub's Team Discussion <sup>[1][github-team-discussion]</sup>
+
+Since "Teams" represent the org's chartered sub-projects and working groups,
+every such group can decide to define its own policy that specifies what they
+consider to be proper use of this feature and enforce that policy internally.
+
+*For General consideration*: The current implementation of GitHub's "Team
+Discussions" is incongruent with the organization's policy. The maximal
+visibility of "public" threads is limited by GitHub to members of the org only,
```

Comment:
```suggestion
visibility of public threads is limited by GitHub to members of the org only,

```

Reply:
But here it's referring to a name of a feature. Not the general idea...

Reply:
i.e. this:
![image](https://user-images.githubusercontent.com/96947/48667600-fc695680-eaa7-11e8-93cc-593a95f89081.png)